src/tests/system/tests/test_ldap_extra_attrs.py

```diff
@@ -42,3 +48,198 @@ def test_ldap_extra_attrs__filled(client: Client, provider: GenericProvider, att
     result = client.tools.getent.passwd("tuser")
     assert result is not None
     assert result.name == "tuser"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_ldap_extra_attrs__check_ldb(client: Client, provider: GenericProvider):
+    """
+    :title: Recently added extra attributes should be in cache db along with their value
+    :setup:
+        1. Create new user "user1"
+        2. Add "firstname:cn, description:gecos, user`s_home_directoy:homeDirectory, user_id:uid, lastname:sn"
+            to ldap_user_extra_attrs
```

Comment:
I don't see `firstname, users_home_directory, lastname` being added in the code below

Reply:
Oh, I forgot to change the docstring when editing the test. Fixed now